va/va.h

```diff
@@ -1294,6 +1294,7 @@ typedef enum
     VAEncMiscParameterBufferType	= 27,
     VAEncMacroblockParameterBufferType	= 28,
     VAEncMacroblockMapBufferType        = 29,
+    VAEncQpBufferType                   = 30,
```

Comment:
How does user use this buffer? Could you document the buffer format in va.h? Otherwise please remove this buffer type.